docker-compose/cassandra/v4/docker-compose.yaml

```diff
@@ -6,6 +6,8 @@ services:
     ports:
       - "9042:9042"
       - "9160:9160"
+    environment:
+      - DB_PASSWORD=${DB_PASSWORD}    
```

Comment:
why `DB_PASSWORD`? Do you have a link to the docs showing that it's a recognized var?